facade.groovy

```diff
@@ -0,0 +1,213 @@
+// We use GrGit for interacting with Git. This gets a hold of it as a dependency like Gradle would
+// TODO: Consider if we should do something to fix/suppress the SLF4J warning that gets logged on first usage?
+@GrabResolver(name = 'jcenter', root = 'http://jcenter.bintray.com/')
+@Grab(group = 'org.ajoberstar', module = 'grgit', version = '1.9.3')
+import org.ajoberstar.grgit.Grgit
+import org.ajoberstar.grgit.Remote
```

Comment:
Hey! I think this import is not used. Please consider removing this. There may be an increase in load times or just straight problems with this.

Reply:
To add onto this, the JsonSlurper remote also isn't used :)